packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -2367,12 +2374,14 @@ const schema: SchemaType<DbPost> = {
     optional: true, nullable: true, hidden: true,
   },
   
+  // This is basically deprecated. We now have them enabled by default
+  // for all users. Leaving this field for legacy reasons.
   sideCommentVisibility: {
     type: String,
     optional: true,
     control: "select",
     group: formGroups.advancedOptions,
-    hidden: (props) => props.eventForm || !userHasSideComments(props.currentUser),
+    hidden: true,
```

Comment:
Was this diff rolled in deliberately?

Reply:
We deliberately decided to remove the option to change side-comment display. But, occurs to me I should inform EA Forum about it.